index.js

```diff
@@ -188,5 +188,12 @@ const saveEditChanges = (event) => {
 
 };
 
+export const valueCheck = (num1, num2) => {
+  if (num1 > num2) {
+    return "Num1 is Bigger then Num2";
+  }
+  return "Num1 is Smaller then Num2";
+}
```

Comment:
**Suggestion:** The function `valueCheck` only checks if `num1` is greater than `num2` but does not handle the case when `num1` is equal to `num2`. Consider adding a condition to handle this scenario. [possible issue]
```suggestion
export const valueCheck = (num1, num2) => {
  if (num1 > num2) {
    return "Num1 is Bigger then Num2";
  } else if (num1 < num2) {
    return "Num1 is Smaller then Num2";
  } else {
    return "Num1 is Equal to Num2";
  }
}
```